### PR TITLE
Gradle project clean up and artifacts with "usi-" prefix

### PR DESCRIPTION
**Pull request by kensipe**
updated gradle project to setup idea as good as I could get.  It now …recognizes the correct projects and builds with a usi prefix

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+    id 'idea'
+}
+
+allprojects {
+
+    version = '0.1'
     apply plugin: "scala"
 
+    targetCompatibility = 1.8
+    compileScala.targetCompatibility = 1.8
```
Comment (jeschkies) [inline build.gradle:14]: Thanks. That broke me when I had Java 11 in my path.

File: settings.gradle [hunk 2]
```diff
@@ -2,4 +2,7 @@ rootProject.name = 'usi'
 
 include 'interfaces'
 include 'core'
-include 'examples:hello-world'
+
+/* example projects*/
+include 'hello-world'
+project(':hello-world').projectDir = new File('examples/hello-world')
```
Comment (jeschkies) [inline settings.gradle:8]: Ah, so this de-references all Gradle project in `./examples`.

**Review by jeschkies — APPROVED** [review]

**Comment by jeschkies** [comment]
Please follow our old commit message format 🙂

File: settings.gradle (hunk 2 above)
Comment (kensipe) [reply settings.gradle:8]: yep yep

**Review by meichstedt — APPROVED** [review]

**Pull request merged**